### DISCUSSION TITLE
4 ideas for features to add

### DISCUSSION THREAD

**Discussion by KRIBEnjoyer** [ShimmyMySherbet/ShimmysAdminTools#25] [Ideas]
Hi, I'd like to suggest 4 features that came to my mind which I think are cool.

- First feature I’d like to see is a blacklist for weapons to which you couldn’t add attachments. For example, so you could block staff from putting attachments on weapons which would become very dangerous if they could add to them. This feature is very similar to the blacklist for attachments themselves.
- Second feature is also related to attachment command. Would be cool if setting a magazine attachment like for example fragmentation rounds would automatically set the amount of the gun’s ammo to the max capacity of the magazine you added to it. This is to remove the need for having unlimited ammo enabled and would also stop abuse of putting something like explosive arrows over full military drum. Additionally, stuff like suppressor attachment when set would automatically get set to 100% quality.
- Third feature is about adding a command which would keep all weapons and suppressors at max quality when enabled. Maybe even add the ability like unlimited ammo command where you can override it with a specified amount.
- Last feature is about adding a configuration option which would allow owners to have the server from start set server jump, speed, and gravity to what ever they want without needing to manually input it every time server restarts.

These are some of the ideas I’d personally like to see but still, even if I don’t, you made an awesome plugin as is and huge props to you. ☺

**Comment by ShimmyMySherbet** [comment]
These all sound like good ideas, so I've created an issue to track progress on this.
See the issue #26
